test/intTest/kip17IntTest.js

```diff
@@ -279,7 +279,7 @@ describe('KIP17 V2 API service', () => {
         caver.initWalletAPI(walletEnv.chainId, walletEnv.accessKeyId, walletEnv.secretAccessKey, walletEnv.url)
     })
 
-    it('CAVERJS-EXT-KAS-INT-247: caver.kas.kip17.deploy should deploy KIP-17 token contract', async () => {
+    it('CAVERJS-EXT-KAS-INT-321: caver.kas.kip17.deploy should deploy KIP-17 token contract', async () => {
```

Comment:
Did you change the testcase no. becuase thease cases were duplicaed?

Reply:
That's right. As a result of reviewing the TC document, I revised it because it was a duplicate number.